.github/workflows/demo.yaml

```diff
@@ -7,6 +7,7 @@ on:
 
 env:
   NODE_VERSION: 14
+  NODE_OPTIONS: --max_old_space_size=4096
```

Comment:
Do we need/want this for all node steps in this file or just the build?

Reply:
It is needed for just the build, but it doesn't hurt to have it everywhere 🤷‍♂️